prettier.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  trailingComma: 'all',
+  trailingComma: 'none',
```

Comment:
For this, let’s just use prettier defaults

Reply:
The default is `"none"` but as there was a config file I just update it. I can just export an empty object then if you like.